src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -72,7 +72,8 @@ public void GenerateServicePageFromTopLevelTOC(TableOfContentsNode node, List<Fi
             if (node.LandingPageType.Value != null)
             {
                 var topLevelTOCRelativeDir = Path.GetDirectoryName(_joinTOCConfig.TopLevelToc);
-                var baseDir = _joinTOCConfig.OutputFolder.IsDefault ? topLevelTOCRelativeDir : _joinTOCConfig.OutputFolder;
+                var referenceTOCRealtiveDir = Path.GetDirectoryName(_joinTOCConfig.ReferenceToc);
```

Comment:
as the spell checker found: realtive -> relative

Reply:
Fixed.